alexandria/core/serializers.py

```diff
@@ -15,7 +15,11 @@
 
 
 def default_user_attribute(user, attribute):
-    return getattr(user, attribute) if not isinstance(user, AnonymousUser) else None
+    return (
+        getattr(user, attribute)
+        if user is not None and not isinstance(user, AnonymousUser)
+        else None
+    )
```

Comment:
Could we do this instead?

```suggestion
    return getattr(user, attribute, None)
```

Reply:
This will also just return `None` if you mistype an attribute on a regular user object, so maybe not the best idea, but we'd fail hard in that case anyway

Reply:
I went a different path. I don't feel like there needs to be a function that does nothing more than `getattr`.